schemas/choice.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://model.choice.json",
+  "title": "CHOICE",
+  "type": "object",
+  "properties": {
+    "locationNodeId": {
+      "description": "The node in which this choice is provided.",
+      "type": "string",
+      "minLength": 5
+    },
+    "destinationNodeId": {
+      "description": "The node to which the current selection leads.",
+      "type": "string",
+      "minLength": 5
+    },
+    "choiceText": {
+      "description": "Description of the essence of the choice in this node.",
+      "type": "string",
+      "minLength": 1
+    },
+    "acceptingStatus": {
```

Comment:
I do not this we should mix model that describes story and saving status, this is really bad. Story is generic for all users, while saving status will be specific for one session of one specific user.

Reply:
`+`